packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -489,6 +489,7 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
       alias,
     },
     resolveLoader: {
+      symlinks: !buildOptions.preserveSymlinks,
```

Comment:
Related: We should check to make sure all loaders are resolved  (`require.resolve`) before use to ensure the proper logic is used.  Especially relevant when peer dependencies are involved but also helpful for eventual pnp support.